src/modules/clarAllTasks.js

```diff
@@ -0,0 +1,22 @@
+const toggleFinishedTask = (e) => {
+  e.nextElementSibling.classList.toggle('line_through');
+  e.nextElementSibling.classList.toggle('change_color');
+  const taskList = JSON.parse(localStorage.getItem('toDoList'));
+  taskList[e.parentElement.id].completed = !taskList[e.parentElement.id].completed;
+  localStorage.setItem('toDoList', JSON.stringify(taskList));
+};
+
+const clearSelectedTasks = () => {
+  let taskList = JSON.parse(localStorage.getItem('toDoList'));
+  taskList = taskList.filter((completed) => completed.completed !== true);
+  let i = 0;
+  taskList.forEach((task) => {
+    task.index = i;
+    // eslint-disable-next-line no-plusplus
+    i++;
+  });
+  localStorage.setItem('toDoList', JSON.stringify(taskList));
+  window.location.reload();
+};
+
```

Comment:
- [ ] As required for the project, kindly make sure when the updated data are stored in local storage, **the user still gets the correct list values after the page reloads**. In your app, if the user reloads the page, it doesn't give the correct data. And some times in your local storage a task can have the completed attributed set to `true` but on the UI the `checkbox` button is not checked. You can test your app to see these issues. Or you can this homage below as an example:

![Screenshot from 2023-01-19 19-37-18](https://user-images.githubusercontent.com/83042742/213519346-6f133b05-f67d-4f03-8b64-b93b855fdb48.png)